packages/react-storage/src/components/StorageBrowser/composables/Pagination.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { CLASS_BASE } from '../views/constants';
+import {
+  ListItemElement,
+  NavElement,
+  OrderedListElement,
+  SpanElement,
+} from '../context/elements';
+import { PaginateButton } from '../components/PaginateButton';
+
+export interface PaginationProps {
+  page: number;
+  hasMorePages: boolean;
```

Comment:
`hasMorePages` should be replaced with `hasNextPage` since that's all we actually know

Reply:
Done